### DISCUSSION TITLE
特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by shunmame** [Yowkees/keyball#333] [Q&A]
写真に赤丸で示したキーが組み立て後反応しませんでした．
はんだの見直し等行ったのですが，変わりませんでした．テンプラしていそうなダイオードの箇所等，原因箇所教えていただけると幸いです．
また，[こちら](https://github.com/Yowkees/keyball/blob/main/keyball39/doc/rev1/buildguide_jp.md#3-7promicro%E3%81%AE%E6%9B%B8%E3%81%8D%E8%BE%BC%E3%81%BF%E3%81%A8%E5%8B%95%E4%BD%9C%E7%A2%BA%E8%AA%8D)で全てのキーの動作を確認したのですが，組み立て後，反応しないということはあるのでしょうか？
よろしくお願いいたします．
![right](https://github.com/Yowkees/keyball/assets/32508729/5ff57e24-2949-4d62-89cd-2e5d09c6c9a7)
![left](https://github.com/Yowkees/keyball/assets/32508729/a068a5d8-e455-4b44-a0e8-556384e93513)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

ピンセットでのキー入力確認ですべてのキーが動いていたようでしたら、ダイオードは問題なく、やはりキーソケットのテンプラが原因だと思います。
キーソケットの左右両端子をピンセットでショートしてみて、入力できないようでしたらテンプラですので、
キーソケットの端子と基板の間に半田がきちんと流れているかご確認いただければと思います。

お手数ですが、ご確認お願いいたします。
